packages/core/admin/admin/src/content-manager/components/DynamicZone/components/Component/index.js

```diff
@@ -147,14 +149,22 @@ const Component = ({
           />
           <AccordionContent>
             <AccordionContentRadius background="neutral0">
-              <FocusTrap onEscape={() => onToggle(index)}>
-                <FieldComponent
-                  componentUid={componentUid}
-                  icon={icon}
-                  name={`${name}.${index}`}
-                  isFromDynamicZone
-                />
-              </FocusTrap>
+              <Suspense
+                fallback={
+                  <Flex justifyContent="center" paddingTop={4} paddingBottom={4}>
+                    <Loader>Loading content.</Loader>
```

Comment:
I didn't find a "loading..." translation key so I copied [this](https://github.com/strapi/strapi/blob/ea72a6e893800ab5513a9b6c3eda0cc4243f0510/packages/core/helper-plugin/lib/src/components/LoadingIndicatorPage/index.js#L21)